### PR TITLE
Migrate visually-hidden-text to Emotion

### LINKED ISSUE #3763: [2/2] Refactor "dynamic styles" and `attrs()` use for Emotion
### Is your feature request related to a problem? Please describe.

In order to migrate to Emotion, we must refactor components that use Styled Components specific APIs:

#### 1. Replace imports to `styled-components` with:
- `@emotion/styled` where the module uses `styled.<tagname>` or `styled(Component)`.
- `@emotion/core` where the module uses `css` tagged template literals.

#### 2. Refactor the use of `.attrs()`:
```js
/**
 * Unlike Styled Components, Emotion does not provide an `attrs()` method to
 * attach props to a component.
 */
const IndexMain = styled.main.attrs({ role: "main" })`
  flex-grow: 1;
`;

/**
 * Instead, they should be attached in the usual way.
 */
const IndexMainWrapper = styled.main`
  flex-grow: 1;
`;

const IndexMain = () => <IndexMainWrapper role="main" />;
```

#### 3. Refactor dynamic styles:

```js
/**
 * In Emotion, functions that are interpolated in css calls will be stringified. 
 */
const Component = styled.div`
  height: ${(props) => props.height};
`;

/**
 * 1. If you want to have a css call based on props, create a function that returns a css call.
 * 2. It can then be called directly with props or interpolated in a styled call. 
 */
const dynamicHeightCss = (props) => css`
  height: ${props.height};
`;

const Component = styled.div`
  ${dynamicHeightCss}
  max-width: 60rem;
`;
``` 

> Instead of thinking in class names...think in terms of styles and composing them together.

https://gist.github.com/simonsinclair/b5f2cd28271161eeb27f98cffb0b523a

### Describe alternatives you've considered
N/A.

### Testing notes
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- ~~[ ] This feature is expected to need manual testing.~~

### Additional context
None.

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves #3763

### CODE REVIEW

**Review by RichardPK — APPROVED** [review]

**Comment by ryanmccombe** [comment]
Screen Shot 2020-06-26 at 16 37 16

**Comment by RichardPK** [comment]
EkHrNdlWAAArCXy

**Review by simonsinclair — APPROVED** [review]

File: packages/components/psammead-visually-hidden-text/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.0.0 | [PR#?](https://github.com/bbc/psammead/pull/?) Migrate to Emotion |
```
Comment (RichardPK) [inline packages/components/psammead-visually-hidden-text/CHANGELOG.md]: wtf RYAN @ryanmccombe

Comment (ryanmccombe) [inline packages/components/psammead-visually-hidden-text/CHANGELOG.md]:
```suggestion
| 2.0.0 | [PR#3873](https://github.com/bbc/psammead/pull/3873) Migrate to Emotion |
```

**Pull request merged**